examples/zephyr/common/net_connect.c

```diff
@@ -48,6 +48,17 @@ void net_connect(void)
 {
     struct net_if *iface = net_if_get_default();
 
+    if (!net_if_is_up(iface))
+    {
+        LOG_INF("Bringing up network interface");
+        int ret = net_if_up(iface);
+        if ((ret < 0) && (ret != -EALREADY))
+        {
+            LOG_ERR("Failed to bring up network interface: %d", ret);
+            return;
+        }
+    }
+
```

Comment:
@mniestroj I would be particularly interested if you have any advice regarding this change. The network interface for the rak5010 doesn't start automatically, I've found I need to call `net_if_up()` where we previously did not. If there is a better approach, I'd be happy to make a change. Thanks!